### PR TITLE
OCPBUGS-32293: Update Jenkins test to use Deployment

### PR DESCRIPTION
**Pull request by apoorvajagtap**
Failing Jenkins Pipeline test blocks multiple PRs from merging. As agreed, skipping these tests temporarily.

### CODE REVIEW

File: test/extended/builds/pipeline_origin_bld.go [hunk 1]
```diff
@@ -91,6 +91,8 @@ var _ = g.Describe("[sig-builds][Feature:JenkinsRHELImagesOnly][Feature:Jenkins]
 		g.Describe("jenkins pipeline build config strategy", func() {
 			g.It("using a jenkins instance launched with the ephemeral template [apigroup:build.openshift.io]", func() {
 				defer cleanup()
+
+				g.Skip("the test failure blocks multiple PRs. As this strategy is deprecated, skipping the test to unblock https://issues.redhat.com/browse/OCPBUGS-32293")
```
Comment (adambkaplan) [inline test/extended/builds/pipeline_origin_bld.go:95]:
```suggestion
				g.Skip("OCPBUGS-32293: temporary skip to unblock build and Jenkins suite tests")
```

**Review by adambkaplan — APPROVED** [review]
/approve

Minor suggestion on the skip message.

**Review by sayan-biswas — APPROVED** [review]

**Comment by adambkaplan** [comment]
:thinking: Now I'm wondering why it took us this long to catch the change...

**Review by soltysh — CHANGES_REQUESTED** [review]
/hold
Where that jenkins-ephemeral is coming from, I checked both https://github.com/openshift/origin/blob/master/examples/jenkins/jenkins-ephemeral-template.json and https://github.com/openshift/origin/blob/master/test/extended/testdata/cmd/test/cmd/testdata/jenkins/jenkins-ephemeral-template.json both of these are creating DCs not deployment? 

Alternatively, if you're changing to a deployment (if the above what I said is wrong), you'll need to update logs command in line 65, it currently requests `logs -f dc/jenkins` which if we're working with a deployment should be `logs -f deploy/jenkins`

**Comment by soltysh** [comment]
Found the culprit in https://github.com/openshift/cluster-samples-operator/pull/540, so yes, aside from that wait, you need to update that run as I mentioned above. 

Also, given we have those two extra places which don't seem to be in-sync with what we have in samples, I'd suggest to sync them as well. To make them all consistent.

**Comment by soltysh** [comment]
/approve cancel
Please ping me directly on slack once all of the above is fixed, I don't want to have a half baked solution.

**Comment by soltysh** [comment]
Also `DumpApplicationPodLogs` works for DCs only, so that needs fixing as well.

**Comment by soltysh** [comment]
Reverting changes to cluster-samples-operator in https://github.com/openshift/cluster-samples-operator/pull/542

**Comment by adambkaplan** [comment]
/close

Replacing with https://github.com/openshift/origin/pull/28732

**Pull request closed without merging**